src/write.c

```diff
@@ -1884,6 +2152,16 @@ avifResult avifEncoderFinish(avifEncoder * encoder, avifRWData * output)
     // Unix epoch.
     uint64_t now = (uint64_t)time(NULL) + 2082844800;
 
+    // -----------------------------------------------------------------------
```

Comment:
Nit: Also move this line into `#if defined(AVIF_ENABLE_EXPERIMENTAL_AVIR)`, or just delete this line.